lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -7,7 +7,7 @@
 %>
 
 <%static:require_module module_name="js/certificates/factories/certificate_whitelist_factory" class_name="CertificateWhitelistFactory">
-        CertificateWhitelistFactory('${certificate_white_list | n, dump_js_escaped_json}', '${generate_certificate_exceptions_url | n, js_escaped_string}', '${certificate_exception_view_url | n, js_escaped_string}', '${generate_bulk_certificate_exceptions_url | n, js_escaped_string}');
+        CertificateWhitelistFactory(${certificate_white_list | n, dump_js_escaped_json}, '${generate_certificate_exceptions_url | n, js_escaped_string}', '${certificate_exception_view_url | n, js_escaped_string}', '${generate_bulk_certificate_exceptions_url | n, js_escaped_string}');
```

Comment:
The method `dump_js_escaped_json` returns an escaped string which is transformed into javascript object using Mako template. That is why I removed `JSON.parse` in `certificate_whitelist_factory.js` because we already have it as a javascript object.